server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -180,15 +197,16 @@ public static void download(
         // We use a dummy stats tracker to ensure the flow doesn't break.
         // TODO: To be revisited as part of https://github.com/opensearch-project/OpenSearch/issues/7567
         RemoteTranslogTransferTracker remoteTranslogTransferTracker = new RemoteTranslogTransferTracker(shardId, 1000);
-        FileTransferTracker fileTransferTracker = new FileTransferTracker(shardId, remoteTranslogTransferTracker);
+        FileTransferTracker fileTransferTracker = new FileTransferTracker(shardId, remoteTranslogTransferTracker, false);
```

Comment:
Are we setting the ckpMetadata flag to false since we dont know where the ckp exists as metadata or not?

Reply:
The download flow code for both the methods (w or w/o md) is same, so this flag's value won't matter during download flow. This purpose of introducing this flag is more during upload flow at one place only. When this flag is true we don't want to mark translog.ckp file transfer status as success.